src/utils/helpers/validations/validations.js

```diff
@@ -1,19 +1,4 @@
-import I18n from "i18n-js";
-
 const ValidationsHelper = {
-  /**
-   * Return the correct validationMessage
-   *
-   * @method validationMessage
-   * @param {String} Overriding validation message
-   * @param {String} i18nString e.g. 'errors.messages.valid'
-   * @param {Object} i18nOptions e.g. { min: 2, max: 8 }
-   * @return {String} message to display
-   */
-  validationMessage: (message, i18nString, i18nOptions = {}) => {
```

Comment:
Please document this as a breaking change.

Reply:
added, thanks